v_4_8_0/files/config/kubelet-worker.yaml.tmpl

```diff
@@ -1,8 +1,6 @@
 kind: KubeletConfiguration
 apiVersion: kubelet.config.k8s.io/v1beta1
-address: ${DEFAULT_IPV4}
 port: 10250
-healthzBindAddress: ${DEFAULT_IPV4}
```

Comment:
Will it be configured by coredns?

Reply:
These are configurations for kubelet and link node ips with kubelet.
They are configurations created per node in the local template but if we want to use the same template here we need to remove these configs and use them as flags becase each node will have his own IP and we cannot define them in the same configmap.

Reply:
I don't get this. Can we create configmap after local file is rendered? So that you don't have complex logic and multiple additional files rendered?

E.g. create configmap here https://github.com/giantswarm/k8scloudconfig/pull/587/files#diff-a503fa1db86f84fce8eb93e5e36bb677R102 instead of k8s-addons?

Reply:
That logic is executed on each node once it gets bootstrapped so we don´t need to create one config map per node, just one configmap for masters and one for workers. 

I liked to put it as an addon because it is executed once one master bootstrap and it is part of the kubernetes config at startup and It will reside in kube-system namespace.

The problem when using the same template for local and configmap is not all values are shared.So if we want to use the same template we need to drop some configs and put them as flags(some implications are here)

Refering to the node patch to use the configmap, we can use the node patch command literally in the service but I saw it very hacky to use something like this instead a template.
```kubectl patch node ${NODE_NAME} -p "{\"spec\":{\"configSource\":{\"configMap\":{\"name\":\"${CONFIG_MAP_NAME}\",\"namespace\":\"kube-system\",\"kubeletConfigKey\":\"kubelet\"}}}}"``` 

But if using this, we remove 2 templates and its rendering command.